chapters/en/chapter6/supplemental_reading.mdx

```diff
@@ -0,0 +1,8 @@
+# Supplemental reading and resources
+
+This unit introduced the text-to-speech task, and covered a lot of ground. 
+Want to learn more? Here you will find additional resources that will help you deepen your understanding of the topics
+and enhance your learning experience.
+
+* [HiFi-GAN: Generative Adversarial Networks for Efficient and High Fidelity Speech Synthesis](https://arxiv.org/pdf/2010.05646.pdf): a paper introducing HiFi-GAN for speech synthesis. 
+* [X-Vectors: Robust DNN Embeddings For Speaker Recognition](https://www.danielpovey.com/files/2018_icassp_xvectors.pdf): a paper introducing X-Vector method for speaker embeddings.
```

Comment:
I'd also add these two below:
1. FastSpeech 2 - https://arxiv.org/abs/2006.04558
2. MQTTS - https://arxiv.org/pdf/2302.04215v1.pdf

This way they get a full-round view of different types of TTS architectures used in the industry.